extras/courses.md

```diff
@@ -8,6 +8,9 @@ or that there was an alternative that fit better into the curriculum.
 
 Courses | Duration | Effort
 :-- | :--: | :--:
+[Learn to Program: The Fundationtals](https://www.coursera.org/learn/learn-to-program) | 7 weeks | 3-4 hours/week
+[Learn to Program: Crafting Quality Code](https://www.coursera.org/learn/program-code) | 5 weeks | 3-4 hours/week
+[Fundamentals of Computing specialization](https://www.coursera.org/specializations/computer-fundamentals) |  6-9 months | 15 hours/weeks
```

Comment:
This is fine, but the "An Introduction to Interactive Programming in Python (Part 1)" is part of this specialization. So I suggest deleting that course if we're going to add this one.

I also request that you capitalize the word "specialization".